gridcapa-swe-runner-app/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -14,6 +14,11 @@
       "name": "swe-commons.zone-id",
       "type": "java.lang.String",
       "description": "Description for swe-commons.zone-id."
+    },
+    {
+      "name": "swe-runner.data-fix.remove-remote-voltage-regulation-in-france",
+      "type": "java.lang.Boolean",
+      "description": "Description for remove-remote-voltage-regulation-in-france."
```

Comment:
Maybe describe this parameter: "Enable or disable the removal of remote voltage regulation in France" for instance

Reply:
This is exactly the reason why I put it as it is, and it is more complicated than that, as it both getSubstation and getCountry returns Optional and not Country directly. The best would be to be able to impelment it directly in powsybl-core and allow to call something like generator.getCountry() that would return an Optional

Reply:
There is no description on others, it could be good to describe them also. But I will include it